src/special/tslint.js

```diff
@@ -1,5 +1,28 @@
-import parseLinter from '../utils/linters';
+import path from 'path';
+import requirePackageName from 'require-package-name';
+import { loadConfig } from '../utils/linters';
+import { wrapToArray } from '../utils/index';
+
+function resolvePresetPackage(preset, rootDir) {
+  if (preset.startsWith('./') || preset.startsWith('../')) {
+    return path.resolve(rootDir, preset);
+  }
+  return preset;
+}
+
+function checkConfig(config, rootDir) {
+  return wrapToArray(config.extends)
+    .filter(preset => !preset.startsWith('tslint:'))
+    .map(preset => resolvePresetPackage(preset, rootDir))
+    .filter(preset => !path.isAbsolute(preset))
+    .map(requirePackageName);
+}
 
 export default function parseTSLint(content, filename, deps, rootDir) {
```

Comment:
Could you please paste this link somewhere around here?

> https://github.com/palantir/tslint/blob/95d9d958833fd9dc0002d18cbe34db20d0fbf437/src/configuration.ts#L289

Reply:
i added jsdoc linking to the docs instead as it explains there in friendlier terms what resolution is used.